core/common/src/main/java/alluxio/util/StopWatch.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.util;
+
+import java.io.Closeable;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A simplified StopWatch implementation which can measure times in nanoseconds.
+ */
+public class StopWatch implements Closeable {
```

Comment:
any specific reason to implement your own `StopWatch`? Is it possible to use either `org.apache.commons.lang.time.StopWatch` or `com.google.common.base.Stopwatch`?

Reply:
@ifcharming Yes,we need some function that org.apache.commons.lang.time.StopWatch does not have.

Reply:
I think com.google.common.base.Stopwatch is enough,and I will delete StopWatch.java